dist/angucomplete-alt.js

```diff
@@ -714,7 +720,7 @@ angular.module('angucomplete-alt').directive('angucompleteAlt', ['$q', '$parse',
     };
 
     var dropdownWasClicked = function dropdownWasClicked() {
-      if (!config.mousedownOn) {
+      if (!config.mousedownOn || !angular.isString(config.mousedownOn)) {
```

Comment:
this this guard still necessary now that it's doing `getAttribute` ? This seems like it would incorrectly return 'dropdownWasClicked = false' if the user clicks an svg icon within the dropdown